.golangci.toml

```diff
@@ -39,6 +39,7 @@
     "scopelint",
     "funlen",
     "bodyclose", # Too many false positives
+    "godox",
```

Comment:
This can be debated but I personally don't see `FIXME`, `NOTE` or `TODO` as something that should trigger an error in the CI. For example here: https://github.com/containous/maesh/pull/439/files#diff-db68cf5cf70ac312d2dbd378e1e18b74R91

Reply:
Okee.